README.md

```diff
@@ -1,2 +1,64 @@
-# devis
-Technical Debt Estimator
+# Devis
+
+Developed for engineering leads, our project provides technical debt information through visual graphs by doing software quality analysis. This allows leads to make sure that their projects are not decaying overtime.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+What things you need to install the software and how to install them
+
+```
+node v12.0.0
+npm 6.9.0
+```
+
+### Installing and Running
+```
+1. node install
+2. npm run dev
+```
+
```

Comment:
We could also include this screenshot so that they can see what the website looks like without setting anything up. 
![devis_fe](https://user-images.githubusercontent.com/13714793/69504957-d016c000-0edb-11ea-8b2e-b5a8a849b390.png)